cli/tests/testdata/test/doc_only.out

```diff
@@ -0,0 +1,5 @@
+Check [WILDCARD]/test/doc_only/mod.ts$2-5.ts
```

Comment:
In case it's not obvious, scenario here is 1 documentation test and 0 executable tests from the same source module.